.ci/lint/lint-r-code.R

```diff
@@ -14,6 +14,7 @@ FILES_TO_LINT <- list.files(
     , recursive = TRUE
     , include.dirs = FALSE
 )
+FILES_TO_LINT <- FILES_TO_LINT[!grepl(paste0("^", file.path(".", ".pixi", "")), FILES_TO_LINT)]
```

Comment:
I don't understand this one. The line above already is saying "only look at files ending in `.r` or `.rmd`" (ignoring casing).... what files like that are in `./.pixi/`?